firmware/cli.c

```diff
@@ -0,0 +1,218 @@
+#include <stdio.h>

```

Comment:
How come this file is not in src folder?

I suggest keep this PR only on lever calibration.
We don't know the root cause of the invalid characters received on Pico 2, so we don't fix it now.

Reply:
oops i put it the wrong directory